phpstan-phpunit-bleeding-edge-override.neon

```diff
@@ -0,0 +1,3 @@
+conditionalTags:
+    PhpParser\NodeVisitor\NodeConnectingVisitor:
+        phpstan.parser.richParserNodeVisitor: true
```

Comment:
Why is this here? How will it be used? Isn't it something we want to move to `phpstan-default.neon`?